lib/acts_as_list/version.rb

```diff
@@ -1,7 +1,7 @@
 module ActiveRecord
   module Acts
     module List
-      VERSION = '0.7.4'
+      VERSION = '0.7.4.1'
```

Comment:
You don't need to increment the version for us. We'll do this as part of the gem release process.